web-common/src/main/java/edu/unc/lib/boxc/web/common/services/PermissionsHelper.java

```diff
@@ -139,6 +145,25 @@ public boolean hasEditAccess(AccessGroupSet principals, ContentObjectRecord meta
         return accessControlService.hasAccess(metadata.getPid(), principals, editDescription);
     }
 
+    /**
+     * True if authenticated permissions are greater than no access
+     * @param metadata
+     * @return
+     */
+    public boolean enhancedAuthenticatedAccess(AccessGroupSet principals, ContentObjectRecord metadata) {
```

Comment:
To stick with naming conventions, this should begin with a verb like`has` or `allow`.

My reading of the method is that it returns true if the Authenticated group has `canViewOriginals`, rather than having permission greater than no access, since neither canViewMetadata or canViewAccessCopies return true. Maybe `allowsFullAuthenticatedAccess` would make sense? Should also update the javadoc